pkg/controller/plan/kubevirt.go

```diff
@@ -459,6 +454,21 @@ func (r *KubeVirt) DataVolumes(vm *plan.VMStatus) (dataVolumes []cdi.DataVolume,
 	return
 }
 
+func (r *KubeVirt) PopulatorVolumes(vmRef ref.Ref) (pvcNames []string, err error) {
+	secret, err := r.ensureSecret(vmRef, r.copyDataFromProviderSecret)
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	annotations := r.vmLabels(vmRef)
+	err = r.createLunDisks(vmRef)
```

Comment:
@liranr23 shouldn't we do it also when not using volume-populators (warm migrations, migrations to remote clusters)?

Reply:
the behavior stays as far i can see. we call it also in dataVolumes function below. but anyway, seems the warm migration needs more work for perfection.

Reply:
ok, and we shall consider placething the call to createLunDisks elsewhere then but that's for a separate PR